### ISSUE TITLE
[Server] As a developer, I should be able to browse a simple static client as part of the server for testing and experimentation

### ISSUE THREAD

**Issue by luke-siedle** [gernzy/gernzy#66]
Gernzy-server requires a very basic static client that uses Laravel's internal views/controllers patterns and leverages the graphql api on the front end, primarily so we can test and experiment with third party libraries and the general flow of the application. 

Dependencies:
1. jQuery for basic dom manipulation
2. jQuery ajax for graphl (look for examples, e.g. https://stackoverflow.com/a/45535690/558287)
3. https://getuikit.com/ for minimal styling on front end.

Initial goals
1. To begin with we'll need a shop home page, served from /shop, which lists all products as a basic implementation. 
2. Get and store the gernzy token in localStorage
3. Ability to add products to cart
4. Ability to view cart
5. Ability to check out and view a form to fill out.
x. Ultimately we'll introduce a payment method (like Stripe), but we can have a separate ticket for this.  

Since we don't have a gernzy admin client yet, we'll just need to use package seeders to create the required data for manual testing. I guess you should only need to seed product data for now. 

https://laravel.com/docs/master/seeding

Maybe have a look at this, I can't find any official docs on seeds within packages. 
https://websanova.com/posts/laravel/creating-a-new-package-in-laravel-5-part-4-database-migrations-and-seeding

**Comment by AlexeiaW** [comment]
So there are a few things to add to the laravel install:

.babelrc file, containing:  `{
    "plugins": ["@babel/plugin-proposal-class-properties"]
}
`

webpack.mix.js file, containing:  `mix.js("resources/js/app.js", "public/js")
    .js("vendor/lab19/cart/src/resources/js/gernzy.js", "public/js")
    .sass("resources/sass/app.scss", "public/css")
    .browserSync("laravel-gernzy.test");
`

and one package in the package.json: "@babel/plugin-proposal-class-properties": "^7.8.0",

So basically edit the .env with your sql config, and then run the seeder I made in src/database/seeds/ProductsSeeder.php which is in the package

Then in the laravel root run 'npm run watch', which will fire up local host, and hot reloading.

**Comment by AlexeiaW** [comment]
@luke-siedle , also for the webpack.mix.js file, replace "laravel-gernzy.test" with your local server alias to the laravel install
